lib/Config.js

```diff
@@ -141,17 +141,20 @@ class Config {
                     assert(typeof lc[l].details.connector === 'object',
                     'bad config: connector must be an object');
                     if (lc[l].details.connector.sproxyd !== undefined) {
-                        const field = sproxydAssert(
+                        this.locationConstraints[l].details.connector.sproxyd =
+                            lc[l].details.connector.sproxyd;
+                        const fields = sproxydAssert(
                             lc[l].details.connector.sproxyd);
-                        if (field === 'chordCos') {
+                        if (fields.indexOf('bootstrap') > -1) {
                             this.locationConstraints[l].details
-                                .connector.sproxyd[field] =
-                                Number.parseInt(lc[l].details.
-                                    connector.sproxyd[field], 16);
-                        } else {
+                                .connector.sproxyd.bootstrap =
+                                lc[l].details.connector.sproxyd.bootstrap;
+                        }
+                        if (fields.indexOf('chordCos') > -1) {
                             this.locationConstraints[l].details
-                                .connector.sproxyd[field] =
-                                lc[l].details.connector.sproxyd[field];
+                                .connector.sproxyd[fields] =
```

Comment:
Is this right? I would expect instead `.connector.sproxyd.chordCos`. From my tests, that works if the array has only one element, but does not as soon as it has 2+.